tox.ini

```diff
@@ -179,3 +179,30 @@ install_command = pip install --no-deps {opts} {packages}
 commands =
     conda list
     {env:PYTEST_COMMAND} {posargs}
+
+[testenv:pyinstaller]
+description = check that sunpy can be included in a pyinstaller bundle
+changedir = .pyinstaller
+deps =
+    # The developer version of PyInstaller doesn't work yet because it
+    # requires https://github.com/pyinstaller/pyinstaller/pull/4888 -
+    # so for now we pin to the latest working commit.
+    git+https://github.com/pyinstaller/pyinstaller.git@c111ec5c#egg=PyInstaller
+    # PyInstaller is not yet compatible with Matplotlib 3.3 so we pin
+    # Matplotlib to an older version for now - see the following issue
+    # for more details: https://github.com/pyinstaller/pyinstaller/issues/5004
+    matplotlib<3.3
+extras =
+    tests
+    net
+    all
```

Comment:
We don't need these here do we? All we are running from tox is pyinstaller? or does pyinstaller copy from your current env?

Reply:
Tox needs these to ensure that the tests run fine by itself, PyInstaller then copies the current environment (tox) and creates the executable based on the environment it was running on.

Reply:
Do we still need all of these now?

all, encompasess net. can we get away with just test?

Reply:
All doesn't seem to load Beautiful Soup which is part of net, this gave me an error locally I might need to check what else is required from tests and add them individually

Reply:
It should, I wonder what the cause is.

Reply:
My apologies, must've been some local issue, we don't need net